internal/server/data/migrations_test.go

```diff
@@ -178,3 +178,21 @@ func tableExists(t *testing.T, db *gorm.DB, name string) bool {
 	assert.NilError(t, err)
 	return count > 0
 }
+
+func TestMigration_AddKindToProvider(t *testing.T) {
```

Comment:
I think `Test202206151027` was already covering this migration for sqlite, but I like this test name better.

Can we keep assertions from that test (the assertion is more strict and tests both cases), but update the test name to this one.